framework/src/modules/validators/endpoint.ts

```diff
@@ -12,6 +12,46 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+import { blsPopVerify } from '@liskhq/lisk-cryptography';
+import { LiskValidationError, validator } from '@liskhq/lisk-validator';
+import { ModuleEndpointContext } from '../..';
 import { BaseEndpoint } from '../base_endpoint';
+import { generatorListSchema, validateBLSKeyRequest, validateBLSKeyRequestSchema } from './schemas';
+import {
+	MODULE_ID_VALIDATORS,
+	STORE_PREFIX_BLS_KEYS,
+	STORE_PREFIX_GENERATOR_LIST,
+} from './constants';
 
-export class ValidatorsEndpoint extends BaseEndpoint {}
+export class ValidatorsEndpoint extends BaseEndpoint {
+	public async getGeneratorList(ctx: ModuleEndpointContext): Promise<Buffer[]> {
```

Comment:
`Buffer[]` is not valid JSON compatible format. it should the response format should be object with only json compatible format like
```
{
  list: string[],
}
```